src/blocks/blockexports.c

```diff
@@ -19,17 +19,21 @@ extern const struct mach_header_64 _mh_execute_header;
 const unsigned char *get_blocks_dat_start(int testnet)
 {
     size_t size;
-    if (testnet)
+    if (testnet == 1)
```

Comment:
This should be using network_type

Reply:
But `network_type` is defined under the namespace `cryptonote` which is a C++ thing, while this file is C. I don't think `network_type` can be used here; it seems inevitable to explicitly put in the integer values of the network types.

Reply:
Ah, good point. Then how about passing int testnet, int devnet ?

Reply:
ok